pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -4,7 +4,7 @@
 from dataclasses import dataclass
 import re
 
-import astroid
+from astroid import nodes
```

Comment:
This was requested by @cdce8p in https://github.com/home-assistant/core/pull/72692#discussion_r884626786
I kept it out originally to reduce the size of that PR.

I think it fits in this PR, being linked to type hints...